python/uv/_find_uv.py

```diff
@@ -16,16 +16,22 @@ def find_uv_bin() -> str:
     targets = [
         # The scripts directory for the current Python
         sysconfig.get_path("scripts"),
-        # The scripts directory for the base prefix (if different)
+        # The scripts directory for the base prefix
         sysconfig.get_path("scripts", vars={"base": sys.base_prefix}),
         # The user scheme scripts directory, e.g., `~/.local/bin`
         sysconfig.get_path("scripts", scheme=_user_scheme()),
-        # Adjacent to the package root, e.g. from, `pip install --target`
-        os.path.join(os.path.dirname(os.path.dirname(__file__)), "bin"),
+        # Above the package root, e.g., from `pip install --prefix`
+        # with module path `<prefix>/lib/python3.13/site-packages/uv`
+        _join(_parents(_module_path(), 3), "bin"),
```

Comment:
```suggestion
        _join(_parents(_module_path(), 4), "bin"),
```
or you'll get `prefix/lib/bin`

Reply:
Thanks!

On the ordering, I agree — but since it's a departure from the existing ordering I think we should do that in a separate pull request.

Reply:
https://github.com/astral-sh/uv/pull/14191